frontend/src/component/common/ConstraintsList/ConstraintItemHeader/ConstraintItemHeader.tsx

```diff
@@ -49,10 +50,15 @@ const CaseSensitive: FC = () => {
 };
 
 const StyledConstraintContainer = styled('div')(({ theme }) => ({
-    display: 'grid',
-    gridTemplateColumns: 'repeat(3, auto)',
-    gap: theme.spacing(2),
-    placeItems: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(1),
+    [theme.breakpoints.up('sm')]: {
```

Comment:
Can we use container queries instead of media queries for these style changes? Lets us focus more on the actual size of the constraint container than on the width of the screen. Especially because the sidebar can expand contract, this might play better.